Postman/Postman.js

```diff
@@ -89,15 +84,14 @@ UserDocument.seriesList[*].lastModified < SeriesDocument.lastModified
 @param { Array } seriesItems- Seriesコレクション
 @return { Array }
 **/
-Postman.prototype._filterSeries = function(user, seriesItems){
-
-	var filteredSeriesItems = seriesItems.map(function(seriesItem){
-		var isBeforeNewRelease = moment(seriesItem.lastModified).isBefore(user.modifiedLog.seriesListAt);
-		if(isBeforeNewRelease){
-			return seriesItem;
-		}
-	});
-	return _.compact(filteredSeriesItems);
+Postman.prototype._filterSeries = function _filterSeries(user, seriesItems) {
+  const filteredSeriesItems = seriesItems.map((seriesItem)=> {
+    const isBeforeNewRelease = moment(seriesItem.lastModified).isBefore(user.modifiedLog.seriesListAt);
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).